test/HostActivationTests/GivenThatICareAboutMultilevelSDKLookup.cs

```diff
@@ -230,7 +230,7 @@ public void SdkLookup_Must_Look_For_Available_Versions_Before_Looking_Into_Anoth
                 .Should()
                 .Pass()
                 .And
-                .HaveStdErrContaining(Path.Combine(_cwdSelectedMessage, "9999.0.0-global-dummy"));
+                .HaveStdErrContaining(Path.Combine(_cwdSelectedMessage, "9999.0.0"));
```

Comment:
Yes, I was planning on adding some more test coverage  - as well as a test for https://github.com/dotnet/core-setup/issues/1760 as I may have fixed it by happenstance...